guides/common/modules/proc_cloning-without-pulp-data.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="cloning-without-pulp-data"]
+= Cloning without Pulp data
+
+You can clone {Project} server without including Pulp data.
+
+.Procedure
+Follow the steps in the procedure in xref:preparing-the-source-server-for-cloning[] and xref:cloning-to-the-target-server[] replace the steps that involve cloning with Pulp data with the following steps:
```

Comment:
I suggest to rephrase this; maybe you can use it as prerequisite. 

Alternative way: You can split sets of steps into snippets and include whatever your need in this and the other procedure. Note that this leads to more duplicate rendered docs.